certbot/plugins/selection.py

```diff
@@ -39,6 +39,35 @@ def pick_authenticator(
     return pick_plugin(
         config, default, plugins, question, (interfaces.IAuthenticator,))
 
+def get_unprepared_installer(config, plugins):
+    """
+    Get an unprepared interfaces.IInstaller object.
+
+    :param certbot.interfaces.IConfig config: Configuration
+    :param certbot.plugins.disco.PluginsRegistry plugins:
+        All plugins registered as entry points.
+
+    :returns: Unprepared installer plugin.
+    :rtype: IPlugin
+    """
+
+    _, req_inst = cli_plugin_requests(config)
+    if not req_inst:
```

Comment:
If you obtain a cert without an installer (e.g. `certonly`) and then run `certbot renew`, the following is output:
```
Could not choose appropriate plugin for updaters: Missing command line flags. For non-interactive execution, you will need to specify an installer plugin on the command line. You can do this by running Certbot with '--installer PLUGIN_NAME
```
We shouldn't do this. How about we just return `None` here?